staging/src/k8s.io/apiserver/plugin/pkg/audit/webhook/webhook_test.go

```diff
@@ -198,10 +198,77 @@ func TestBatchWebhookStopCh(t *testing.T) {
 		waitForEmptyBuffer(backend)
 		close(stopCh) // stop channel has stopped
 	}()
-	backend.sendBatchEvents(stopCh, timer)
+	backend.sendBatchEvents(backend.collectEvents(stopCh, timer))
 	require.Equal(t, expected, <-got, "get queued events after timer expires")
 }
 
+func TestBatchWebhookProcessEventsAfterStop(t *testing.T) {
+	events := make([]*auditinternal.Event, 1) // less than max size.
```

Comment:
Because it's consistent with the rest of the code, that way it's clear how many events there are and you can easily change